Assets/Scripts/Weapons/BaseWeapon.cs

```diff
@@ -0,0 +1,61 @@
+using Mirror;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class BaseWeapon : NetworkBehaviour
+{ 
+    public float coolDownTime;      // Minimal time between weapon uses in seconds
+    public float projectileSpeed;
+    public int damage;
+
+	protected GameObject projectilePrefab;
+
+	private Transform playerTransform;
```

Comment:
```suggestion
    protected GameObject projectilePrefab;

    private Transform playerTransform;
```
Podobnie tutaj ze spacjami

Reply:
Done